docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+**Optional flags**:
+- `-d` - If you want to run the server in DEBUG mode which facilitates problem diagnosis, you can include the `-d` flag on the start command. However, in general it is not recommended to run a server with debug enabled as this will cause the server to perform slower.
+- `-p` - If you want the server to run on a port different than what is specified in the configuration .yaml file, you can override the existing port.
+
+### Enroll bootstrap user with TLS CA
+
+Now that your TLS CA is configured and before you can deploy any other nodes for your organization, you need to enroll the bootstrap user of the node to generate the TLS certificates. Since the CA server is up and running, instead of using the **Fabric CA server CLI commands** we now use the **Fabric CA client CLI commands** to submit an enrollment request to the server. Before you can deploy another new CA with TLS enabled, you need to provide the new CA TLS certificate and private key to the configuration which allows the CA to transact securely with the other nodes in the network. The following process is described for a CA node, but can be easily extended for peer or ordering nodes.
+
+Performed by using the Fabric CA client, the enrollment process is used to generate the certificate and private key pair which forms the node TLS identity. You should have already setup the required folders in the [Fabric CA client](#fabric-ca-client) section.
+
+The folder structure we are using for these commands is
+  ```
+  fabric-ca-client
+    └── tls-ca
+      ├── admin
+      └── tls
+  ```
+  These folders are used by the Fabric CA client to:
+   - Store the certificates that are issued when the Fabric CA client enroll command is run against the TLS CA server. (**admin** folder)
+   -  Know where the TLS certificates reside that allows the Fabric CA client to communicate with the TLS CA server. (**tls** folder)
+
+1. Copy the TLS CA root certificate file `fabric-ca-server-tls/ca-cert.pem`, that was generated when the TLS CA server was started, to the folder `fabric-ca-client/tls-ca/tls/tls-ca-cert.pem`. Notice the file name is changed to `tls-ca-cert.pem` to make it clear this is the root certificate from the TLS CA. **Important:** This TLS CA root certificate will need to be available on each host that will run commands against the TLS CA.
+
+2. The Fabric CA Client also needs to know where to put the certificates that are generated by the `enroll` command. The destination is controlled by the value of the `FABRIC_CA_CLIENT_HOME` environment variable.  Modify the `FABRIC_CA_CLIENT_HOME` to point to the `/admin` folder that was created under the `tls-ca` folder above:
```

Comment:
I'm not understanding why HOME directory would be /admin.

Reply:
I'm following the same pattern described in the [Fabric CA operations guide](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-tls-ca-s-admin).

The Fabric CA client needs to know where to put the certificates that are generated by the `enroll` command. The way the CLI works is that the destination is controlled by the value of the `FABRIC_CA_CLIENT_HOME` environment variable.  

Since these are the admin certs for the TLS CA bootstrap identity, they go in the admin folder.

Reply:
I've completed refactored the folder structure to use the FABRIC_CA_CLIENT_HOME environment variable to designate only the location of the binary file.